src/@types/index.d.ts

```diff
@@ -6,19 +6,34 @@ interface QueryParameters {
   [key: string]: string;
 }
 
+//  Classifiers
+// ---------------------------------------------------------------------------
+type CountryCode = string;
+type ASN = number;
+type ConnectionType = string;
+type DeviceType = string;
+
 //  Server response
 // ---------------------------------------------------------------------------
 
 interface Test {
   id: string;
 }
 
-interface Client {
+interface Agent {
   hasFeatureSupport: boolean;
 }
 
+interface Client {
+  country_code: CountryCode;
```

Comment:
does eslint not check this file? Surprised you didn't need `/* eslint-disable @typescript-eslint/camelcase */` here.